config/crd/kustomizeconfig.yaml

```diff
@@ -0,0 +1,19 @@
+# This file is for teaching kustomize how to substitute name and namespace reference in CRD
```

Comment:
do we use this?

Reply:
[kustomization.yaml](https://github.com/IBM/ibm-block-csi-operator/blob/CSI-3123/Uograde_operator_sdk_to_1.9.0/config/crd/kustomization.yaml) is using it and we are using [kustomization.yaml](https://github.com/IBM/ibm-block-csi-operator/blob/CSI-3123/Uograde_operator_sdk_to_1.9.0/config/crd/kustomization.yaml)

Reply:
where exactly in the yaml is this `name and namespace` part?

Reply:
removed it